src/main/scala/com/mozilla/telemetry/streaming/EventsToAmplitude.scala

```diff
@@ -44,57 +69,49 @@ object EventsToAmplitude {
   val writeMode = "error"
 
   private class Opts(args: Array[String]) extends ScallopConf(args) {
+    val configFilePath: ScallopOption[String] = opt[String](
```

Comment:
Are these files in a repo somewhere? If not, I feel like they should be (and, tbh, I think this repo is a pretty good place for it) and, if so, we should probably add some documentation/comments linking the two together

Reply:
I was deciding where they should live. I'll stick it in here for now, and we may change it later.